n3fit/src/n3fit/tests/test_preprocessing.py

```diff
@@ -0,0 +1,29 @@
+from n3fit.layers import Preprocessing
+import tensorflow as tf
+import numpy as np
+
+
+def test_preprocessing():
+    """Regression test"""
+    # taken from basic runcard
+    flav_info = [
+        {'fl': 'sng', 'smallx': [1.05, 1.19], 'largex': [1.47, 2.7]},  # 'trainable': False},
+        {'fl': 'g',   'smallx': [0.94, 1.25], 'largex': [0.11, 5.87]},  # 'trainable': False},
+        {'fl': 'v',   'smallx': [0.54, 0.75], 'largex': [1.15, 2.76]},  # 'trainable': False},
```

Comment:
```suggestion
        {'fl': 'sng', 'smallx': [1.05, 1.19], 'largex': [1.47, 2.7], 'trainable': False},
        {'fl': 'g',   'smallx': [0.94, 1.25], 'largex': [0.11, 5.87], 'trainable': False},
        {'fl': 'v',   'smallx': [0.54, 0.75], 'largex': [1.15, 2.76], 'trainable': False},
```
Doesn't really change anything, but since we want to keep supporting setting trainable per flavor this should never give an error.